macosx/Controller.mm

```diff
@@ -4772,6 +4772,11 @@ - (BOOL)validateMenuItem:(NSMenuItem*)menuItem
     {
         BOOL warning = NO;
 
+        if (self.fFilterBar.isFocused == NO)
```

Comment:
This disables the delete action whether the search field is focused or not.

Reply:
> Thanks for the initial attempt, but it causes a regression: cmd+delete will stop working even after selecting a row explicitly.

Sorry. Fixed.